docs/install_guides/installing-openvino-conda.md

```diff
@@ -1,26 +1,39 @@
+This guide provides installation steps for Intel® Distribution of OpenVINO™ toolkit through the Anaconda Cloud.
 
 > **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. If you want to develop or optimize your models with OpenVINO, see [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
 
 ## System Requirements
 
-**Software**
+@sphinxdirective
 
- - [Anaconda distribution](https://www.anaconda.com/products/individual/)
+.. tab:: Anaconda Software
 
-**Operating Systems**
+   [Anaconda distribution](https://www.anaconda.com/products/individual/)
 
-| Supported Operating System                                   | [Python Version (64-bit)](https://www.python.org/) |
-| :------------------------------------------------------------| :---------------------------------------------------|
-|   Ubuntu 18.04 long-term support (LTS), 64-bit              | 3.6, 3.7, 3.8, 3.9                                  |
-|   Ubuntu 20.04 long-term support (LTS), 64-bit              | 3.6, 3.7, 3.8, 3.9                                  |
-|   Red Hat Enterprise Linux 8, 64-bit                         | 3.6, 3.7, 3.8, 3.9                                  |
-|   macOS 10.15                                             | 3.6, 3.7, 3.8, 3.9                                  |
-|   Windows 10, 64-bit                                        | 3.6, 3.7, 3.8, 3.9                                  |
+.. tab:: Supported Operating Systems with Python Version
 
-## Install OpenVINO Runtime Using the Anaconda Package Manager
+   +--------------------------------------+----------------------------------------------------+
+   | Operating System (64-bit)            | [Python Version (64-bit)](https://www.python.org/) |
+   +======================================+====================================================+
+   | macOS 10.15                          | 3.6, 3.7, 3.8, 3.9                                 |
+   +--------------------------------------+----------------------------------------------------+
+   | Red Hat Enterprise Linux 8           | 3.6, 3.7, 3.8, 3.9                                 |
+   +--------------------------------------+----------------------------------------------------+
+   | Ubuntu 18.04 long-term support (LTS) | 3.6, 3.7, 3.8, 3.9                                 |
+   +--------------------------------------+----------------------------------------------------+
+   | Ubuntu 20.04 long-term support (LTS) | 3.6, 3.7, 3.8, 3.9                                 |
+   +--------------------------------------+----------------------------------------------------+
+   | Windows 10                           | 3.6, 3.7, 3.8, 3.9                                 |
+   +--------------------------------------+----------------------------------------------------+  
+
+
+@endsphinxdirective
+
+## Installation Flow
```

Comment:
This heading is not necessary, because the following section directly introduces the steps instead of having an overview of the installation flow.

Reply:
I've reverted it, as you requested.